in-work/lln_clt.md

```diff
@@ -42,7 +37,6 @@ import scipy.stats as st
 
 ## Relationships
 
-
 The LLN gives conditions under which sample moments converge to population moments as sample size increases.
```

Comment:
```suggestion
The LLN gives conditions under which sample moments converge to population moments as sample size increases.
```

Great work @HumphreyYang!

I just have a suggestion that starting off with sample and population might be a little confusing since its an introductory lecture. Maybe a less technical explanation like "LLN states that if you repeat an experiment independently a large number of times and average the result, what you obtain should be close to the expected value" could be used to start with. And the same for CLT.

Reply:
thanks @maanasee